program.js

```diff
@@ -0,0 +1,63 @@
+var buttons = document.querySelector("#submi");
+buttons.addEventListener('click', changesome);
+
+function changesome() {
```

Comment:
You could give your function a less general name.

A good function name describes what it is that your function does. You could summarise the changes that happen to your page when this function is called by your event listener.